go/teams/service_helper.go

```diff
@@ -648,22 +648,35 @@ func AddEmailsBulk(ctx context.Context, g *libkb.GlobalContext, teamname, emails
 
 func EditMember(ctx context.Context, g *libkb.GlobalContext, teamname, username string,
 	role keybase1.TeamRole, botSettings *keybase1.TeamBotSettings) error {
+	team, err := Load(ctx, g, keybase1.LoadTeamArg{
+		Name:        teamname,
+		ForceRepoll: true,
```

Comment:
This adds one extra RPC to EditMember. It would be possible to avoid by for example passing `*Team` down into helpers after it's loaded.

Reply:
will address in the next PR and drop the extra loads with new RPCs for impteam changes